pkg/crypto/key_pair_secrets.go

```diff
@@ -109,6 +109,7 @@ func NewKeyPairSecrets(seed []byte, path string, seedMethod SeedMethod, password
 
 // PathForDIDType Build a valid path string given didType and name string
 func PathForDIDType(name string, didType id.DidType) string {
+	// todo: add validation here. Name should not be "".
```

Comment:
TODO for now or for another ticket (+ other places)?

Reply:
I'll remove the comment incase anyone used it with "" which will work. I guess its a name and not a security issue.